tests/e2e/run-scanner-v4.bats

```diff
@@ -56,8 +56,13 @@ setup() {
 
     if (( test_case_no == 0 )); then
         # executing initial teardown to begin test execution in a well-defined state
-       run remove_existing_stackrox_resources
+        teardown
     fi
```

Comment:
There is function `setup_file()` that is run at the beginning of all tests in the file. Maybe that would be a better solution than counting the tests?

Reply:
Thanks. I believe I had considered this, but it didn't really work out as planned. I will revisit this in a follow-up PR for restructuring and simplifying this test suite, ok?